attribute-service-impl/src/test/java/org/hypertrace/core/attribute/service/AttributeServiceImplTest.java

```diff
@@ -295,6 +296,97 @@ public void testFindAttributes() {
     }
   }
 
+  @Test
+  public void testGetAttributes() {
```

Comment:
I think this test is quite long and I think it is because the objects' creation is very long. I am not sure every single attribute in the objects is relevant for the test, How about abstracting them in private statics and make this test focus on what matters?

Reply:
Yeah, since this is a new api over an existing set of functionality, I didn't write this test from scratch. I took the test from the old api (which was significantly _more_ verbose, if you can imagine) and used it again here to make sure they behaved consistently. I can refactor those down a bit.